app/classes/aapbRecordHelpers.tsx

```diff
@@ -275,9 +283,10 @@ export class AAPBRecord extends Component<AAPBRecordProps> {
         if (embedPlayer) {
           recordBlock = (
             <div
-              style={{ backgroundImage: thumbnail }}
+              style={{ backgroundImage }}
```

Comment:
I think this needs to be {{ backgroundImage: backgroundImage }}

Reply:
Yep!